src/js/api/local-storage-api.js

```diff
@@ -160,7 +158,7 @@ export default class LocalStorage {
         return this._data.news.filter(value => value.isFavorite === true);
     }
 
-    getRead() {
+    getReaded() {
```

Comment:
Функція має називатися getRead. 
```suggestion
    getRead() {
```